.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 python: "3.5"
 install:
   - pip install -r requirements.txt
+  - pip install -r requirements-dev.txt
```

Comment:
Please, in commit message specify that dependencies will be installed both from `requirements.txt` and  `requirements-dev.txt`.